tools/mmp/driver.cs

```diff
@@ -1685,8 +1712,19 @@ static void ProcessNativeLibrary (HashSet<string> processed, string library, Lis
 				// install_name_tool gets angry if you copy in a read only native library
 				CopyFileAndRemoveReadOnly (real_src, dest);
 			}
-
+			
 			bool isStaticLib = real_src.EndsWith (".a", StringComparison.Ordinal);
+			bool isDynamicLib = real_src.EndsWith (".dylib", StringComparison.Ordinal);
```

Comment:
what happens for frameworks ?

Reply:
Nothing, but something very likely should. Let me get someone in to test this case in submission to Apple.